devel/jsonnet/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        google jsonnet 0.14.0 v
+revision            0
+categories          devel
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         The data templating language
+
+long_description    A data templating language for app and tool developers
+
+homepage            http://jsonnet.org/
+
+checksums           rmd160  2e0b39c5070fc02c9b645e2e638a516d0effe44a \
+                    sha256  3521b8fa65c4cf06c1ff7daba5792304fadc87aa640cbdcb613fbe2fecaed0fb \
+                    size    21862029
+
+compiler.cxx_standard 2011
+
+use_configure       no
```

Comment:
You need to make sure you are [using the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler) and test or disable the `universal` variant.

Reply:
Yes, this is necessary, otherwise the build fails. Discussed on the dev mailing list.

Reply:
It's not about the build failing or not. This means that Macports selected compiler usually doe snot get set properly and you need to make sure it does. Also, you have not addressed the `universal` variant. I don't see how it would work in that case.

Reply:
Can you post a link to the discussion?

Reply:
https://lists.macports.org/pipermail/macports-dev/2019-November/041518.html

Reply:
I don’t know too much about MacPorts compilers or the `universal` variant so if you just tell me the right settings I’ll toss them in. I do believe based on the 🤗 transformers port dependencies that Google likes cxx11 so it wasn’t a surprise that jsonnet needs that here.

Reply:
I don't know the right settings to fix the universal variant because it is dependent on this port build system. You have to build it and look at the errors and fix them. Otherwise you should at least turn off the non-working variant. 

`universal_variant no
`

Reply:
Looking briefly at the thread, it is way more time and knowledge than I have to determine if what you are doing is the right solution. It appears that the thread ends without resolution.

Reply:
> It appears that the thread ends without resolution.

The build successfully compiles a working binary and libraries, so afaik it’s resolved.

Reply:
Note: this port select causes the build to fail:
```
port select --show clang
The currently selected version for 'clang' is 'mp-clang-9.0'. 
port select --show gcc
The currently selected version for 'clang' is 'mp-gcc-9. 
```

Setting clang and gcc to `none` works.

What’s the correct blacklist invocation?

Reply:
The fact that `port select` is affecting the build proves that it is not [using the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler).

Reply:
The latest versions build without the compiler directive.